<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDialog</class>
 <widget class="QDialog" name="ClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Données du client</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>471</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="3">
     <widget class="QLineEdit" name="villelineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="commentairelabel">
      <property name="text">
       <string>Commentaire</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="villelabel">
      <property name="text">
       <string>Ville*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="adrlabel">
      <property name="text">
       <string>Adresse*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="jourlabel">
      <property name="text">
       <string>Jour*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nomlabel">
      <property name="text">
       <string>Nom*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="dureelabel">
      <property name="text">
       <string>Durée(mins)*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QDateEdit" name="jourEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QComboBox" name="priopriteBox">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="tellabel">
      <property name="text">
       <string>Tel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="codelabel">
      <property name="text">
       <string>Code postal*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="prenomlabel">
      <property name="text">
       <string>Prénom*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="priolabel">
      <property name="text">
       <string>Priorité*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLineEdit" name="dureelineEdit">
      <property name="text">
       <string>60</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="codelineEdit"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="adrlineEdit"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="tellineEdit"/>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="prenomlineEdit"/>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="nomlineEdit"/>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QTextEdit" name="commentaireEdit"/>
    </item>
    <item row="4" column="7">
     <widget class="QListView" name="RessourcelistView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="reslabel">
      <property name="text">
       <string>Ressource*</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="diverslabel">
      <property name="text">
       <string>Divers</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QTextEdit" name="diversEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ClientAjouterPB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClientAnnulerPB">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exporter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ClientAnnulerPB</sender>
   <signal>clicked()</signal>
   <receiver>ClientDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
